src/main/kotlin/controllers/worldRenderer/RuneliteRenderer.kt

```diff
@@ -71,12 +71,14 @@ import org.lwjgl.opengl.GL30C.glUniform1ui
 import org.lwjgl.opengl.GL31C.GL_UNIFORM_BUFFER
 import org.lwjgl.opengl.GL31C.glGetUniformBlockIndex
 import org.lwjgl.opengl.GL31C.glUniformBlockBinding
+import java.awt.Color
 import java.nio.ByteBuffer
 import kotlin.math.min
 
 // Renderer code which appears to have originated with RuneLite
 open class RuneliteRenderer(
     private val textureManager: TextureManager,
+    var skyColor: Color,
```

Comment:
Should probably be abstract and overridden in the subclass, rather than passed as a value